doc/lua_api.txt

```diff
@@ -2590,6 +2591,37 @@ An `InvRef` is a reference to an inventory.
 * `get_location()`: returns a location compatible to `minetest.get_inventory(location)`
     * returns `{type="undefined"}` in case location is not known
 
+### `AreaStore`
+A fast access data structure to store areas, and find areas near a given position or area.
+Every area has a `data` string attribute to store additional information.
+You can create an empty `AreaStore` by calling `AreaStore()`, or `AreaStore(type_name)`.
+If you chose the parameter-less constructor, a fast implementation will be automatically chosen for you.
+#### Methods
+* `get_area(id, borders, data)`: returns the area with the id `id`. (optional) Boolean values `borders` and `data` control what's copied.
```

Comment:
Does this return a table like `{id=1,min={},max={},data=""}`?  You should define what an "area" is.